core/src/main/java/com/graphhopper/reader/dem/GMTEDProvider.java

```diff
@@ -0,0 +1,261 @@
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Elevation data from Global Multi-resolution Terrain Elevation Data 2010 (GMTED2010).
+ * The data provides 7.5 arc seconds resolution (~250 m) global elevation data. The data is available between latitudes
+ * of 84°N and 70°S. The data is available as .tiff and the we are using the mean elevation per cell (other options are
+ * median, min, max, etc.).
+ * <p>
+ * More information can be found here: https://topotools.cr.usgs.gov/gmted_viewer/
+ * <p>
+ * When using the data we have to acknowledge the source: "Data available from the U.S. Geological Survey.",
+ * more information can be found here: https://lta.cr.usgs.gov/citation
+ * <p>
+ * The gdalinfo of one GeoTiff is:
+ * Driver: GTiff/GeoTIFF
+ * Files: 50N000E_20101117_gmted_mea075.tif
+ * Size is 14400, 9600
+ * Coordinate System is:
+ * GEOGCS["WGS 84",
+ * DATUM["WGS_1984",
+ * SPHEROID["WGS 84",6378137,298.257223563,
+ * AUTHORITY["EPSG","7030"]],
+ * AUTHORITY["EPSG","6326"]],
+ * PRIMEM["Greenwich",0],
+ * UNIT["degree",0.0174532925199433],
+ * AUTHORITY["EPSG","4326"]]
+ * Origin = (-0.000138888888889,69.999861111111116)
+ * Pixel Size = (0.002083333333333,-0.002083333333333)
+ * Metadata:
+ * AREA_OR_POINT=Area
+ * Image Structure Metadata:
+ * INTERLEAVE=BAND
+ * Corner Coordinates:
+ * Upper Left  (  -0.0001389,  69.9998611) (  0d 0' 0.50"W, 69d59'59.50"N)
+ * Lower Left  (  -0.0001389,  49.9998611) (  0d 0' 0.50"W, 49d59'59.50"N)
+ * Upper Right (  29.9998611,  69.9998611) ( 29d59'59.50"E, 69d59'59.50"N)
+ * Lower Right (  29.9998611,  49.9998611) ( 29d59'59.50"E, 49d59'59.50"N)
+ * Center      (  14.9998611,  59.9998611) ( 14d59'59.50"E, 59d59'59.50"N)
+ * Band 1 Block=14400x1 Type=Int16, ColorInterp=Gray
+ * Min=-209.000 Max=2437.000
+ * Minimum=-209.000, Maximum=2437.000, Mean=149.447, StdDev=239.767
+ * NoData Value=-32768
+ * Metadata:
+ * STATISTICS_EXCLUDEDVALUES=-32768
+ * STATISTICS_MAXIMUM=2437
+ * STATISTICS_MEAN=149.44718774595
+ * STATISTICS_MINIMUM=-209
+ * STATISTICS_STDDEV=239.767158482
+ *
+ * @author Robin Boldt
+ */
+public class GMTEDProvider extends AbstractTiffElevationProvider {
+    private static final int WIDTH = 14400;
+    private static final int HEIGHT = 9600;
+    // TODO is the precision correct?
+    private final double precision = 1e7;
+    private final int latDegree = 20;
+    private final int lonDegree = 30;
+    // for alternatives see #346
+    private final String FILE_NAME_END = "_20101117_gmted_mea075";
+
+    public GMTEDProvider() {
+        super("https://edcintl.cr.usgs.gov/downloads/sciweb1/shared/topo/downloads/GMTED/Global_tiles_GMTED/075darcsec/mea/",
+                "/tmp/gmted",
+                "GraphHopper GMTEDReader");
+    }
+
+    public static void main(String[] args) {
+        GMTEDProvider provider = new GMTEDProvider();
+
+        System.out.println(provider.getEle(46, -20));
+
+        // 337.0 (339)
```

Comment:
The first number is the SRTM/CGIAR result for this coordinate, the result of GMTED is in braces.